src/pudl/extract/epacems.py

```diff
@@ -36,7 +36,7 @@
     # "SO2_RATE (lbs/mmBtu)": "so2_rate_lbs_mmbtu",  # Not reading from CSV
     # "SO2_RATE": "so2_rate_lbs_mmbtu",  # Not reading from CSV
     # "SO2_RATE_MEASURE_FLG": "so2_rate_measure_flg",  # Not reading from CSV
-    "NOX_RATE (lbs/mmBtu)": "nox_rate_lbs_mmbtu",
+    # "NOX_RATE (lbs/mmBtu)": "nox_rate_lbs_mmbtu",
     # "NOX_RATE": "nox_rate_lbs_mmbtu",  # Not reading from CSV
     # "NOX_RATE_MEASURE_FLG": "nox_rate_measurement_code",  # Not reading from CSV
```

Comment:
Don't we need all of these (including the multiple different column names) if one ever wanted to read those columns out of the raw data? It seems like leaving the mappings here, and then including them as appropriate in the list of columns to ignore would be better.  Especially knowing that a single column has different names in different years of data is useful bespoke knowledge.

Reply:
ohhh ok I was wondering why there were two ways to ignore a column. This makes a lot more sense. However, isn't the idea that rates can be calculated therefore we don't want to include them? Or maybe we should in the raw simply because they are there and there could be interesting calculation errors. Either way, we should probably keep all the rate columns or none in the raw, not just the NOX ones (unless there's a specific reason).